sound/soc/sof/topology.c

```diff
@@ -894,16 +861,19 @@ static int sof_parse_word_tokens(struct snd_soc_component *scomp,
 			found++;
 		}
 	}
+
+	return found;
 }
 
-static int sof_parse_tokens(struct snd_soc_component *scomp,
-			    void *object,
-			    const struct sof_topology_token *tokens,
-			    int count,
-			    struct snd_soc_tplg_vendor_array *array,
-			    int priv_size)
+static int sof_parse_token_sets(struct snd_soc_component *scomp,
```

Comment:
I think this function could benefit from a comment, explaining parameters, e.g. that each of `sets` sets has `count` elements with the total size of a set equal to `objectsize`?

Reply:
ok